### ISSUE TITLE
[🐛 Bug]: Localhoat isn't Working !

### ISSUE THREAD

**Issue by SutharShantanu** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#777]
### What went wrong?

![Screenshot (14)](https://user-images.githubusercontent.com/110021464/190199952-3a3534f1-775a-4886-b6f8-0ba5a26628d0.png)

see I am unable to get a token.

### What did you expect to happen?

expected to be local host give me the token then I will paste it in google token fill feild 

### Steps to reproduce the bug

1. As soon you Choose JSON file.
2. Follow step till the browser open and ask google authentication.
3. after Pressing Continue & giving confirmation to access the user account.
4. there I face a local host page showing the error "This site can’t be reached".

### App Version

v1.0.0

### OS Version

Windows 11 Pro 22H2  OS Build 22621.457

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%F0%9F%90%9B+bug%22) and didn't find any duplicate issue
- [X] I have checked the [Frequently Asked Questions (FAQs)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ)) and can confirm that there's no fixes or workarounds for this issue
- [X] This issue only covers one particular bug or issue

**Comment by im-coder-lg** [comment]
Copy the ?code query from the url and you are good to go. Only copy that query; ***omit*** the others.

**Comment by SutharShantanu** [comment]
> Copy the ?code query from the url and you are good to go. Only copy that query; _**omit**_ the others.

I Apologies if I Misunderstood you.
I guess you are asking for LocalHost URL "https://localhost:5754/auth/handler?code=4/0ARtbsJqmsymDt65kx-NMj9kGHWo5ALkWH-k2qhqu75aW3iCKTXP1bABRf-c0ChVi2YdXtQ&scope=https://www.googleapis.com/auth/assistant-sdk-prototype"

**Comment by im-coder-lg** [comment]
Yeah just copy the contents between the ?code= till the next question mark, then you are good to go.

**Comment by SutharShantanu** [comment]
> Yeah just copy the contents between the ?code= till the next question mark, then you are good to go.

I guess this is "4/0ARtbsJqmsymDt65kx-NMj9kGHWo5ALkWH-k2qhqu75aW3iCKTXP1bABRf-c0ChVi2YdXtQ&"

**Comment by im-coder-lg** [comment]
Yep, just put that.

**Comment by SutharShantanu** [comment]
> 4/0ARtbsJqmsymDt65kx-NMj9kGHWo5ALkWH-k2qhqu75aW3iCKTXP1bABRf-c0ChVi2YdXtQ&

It worked! 😲
Thanks a lot for your Valuable Time.
